Composer/packages/client/src/CreationFlow/CreateOptions/index.js

```diff
@@ -72,10 +72,10 @@ export function CreateOptions(props) {
 
   const handleJumpToNext = () => {
     if (option === 'Create from template') {
-      if (template === null) onNext(templates[0]);
+      if (template === null) onNext(templates[1].key);
```

Comment:
let's make it clear by not using a magic number. 

probably explicitly say `emptyBotId = templates[1].key`

Reply:
And please investigate, when this template is null, why not we default having a choice set for user.